### ISSUE TITLE
MVSEC outdoor data parameters

### ISSUE THREAD

**Issue by ahadviger** [HKUST-Aerial-Robotics/ESVO#5]
Hello,

I am trying to run ESVO on outdoor driving data (outdoor_day1 or outdoor_day2) from the MVSEC dataset. I modified the rosbags as suggested and slowed down playback. Unfortunately, I have had no luck with tuning the parameters to get satisfying results. I can barely get 150-200 initialization points from SGM, which is likely not enough for reliable performance. Do you have any suggestions for tuning the parameters for this scenario?

Thanks!

**Comment by knelk** [comment]
Dear ahadviger,

have you found good parameters and would like to share it here please? 

This could be relevant for many other users as well.

Thanks!

**Comment by ahadviger** [comment]
Hi @knelk,

unfortunately I did not find parameters that would significantly improve performance of the method, but what I think helped the most was tuning `invDepth_min_range` and `invDepth_max_range` to allow for greater depths. I suppose that small resolution and lack of events in the great part of the view (road, car hood) are the main reason for inferior performance compared to indoor sequences. Nevertheless, please let me know if you manage to figure something out!

**Comment by Ethan-Zhou** [comment]
Hi @ahadviger,
I recently manage to look into this issue. In short, the current ESVO pipeline cannot get consistent trajectory estimation and depth map on Upenn_driving_outdoor dataset. The reasons consist of:
1) The stereo baseline is too short if compared to the range of the scene.
2) The resolution (spatial) of the sensor is limited (only 346 x 260). It's hard to obtain a high-quality depth map from these coarse observations in a large scale environment. However, we witnessed reasonably good results (attached below) when structures were close to the cameras.

![Upenn_outdoor_issue](https://user-images.githubusercontent.com/5391344/108971304-2402ec80-76bd-11eb-8aab-c7d75f89139c.png)

Regarding parameters used, your intuition was correct. I just modified the invDepth's range properly and increased the number of fusions. That's it. Thanks for your effort on investigating this anyway.

Cheers,

yi
